src/renderer/scss/_vars.scss

```diff
@@ -6,6 +6,8 @@ $width-page-constrained: 800px;
 $text-color: #000;
```

Comment:
ignore the weird stuff going on here. still need to determine proper colors to use